forgerock-openbanking-aspsp/forgerock-openbanking-aspsp-rs/forgerock-openbanking-rs-ui/src/main/java/com/forgerock/openbanking/rs/ui/ForgerockOpenbankingRsUiApplication.java

```diff
@@ -155,6 +167,11 @@ protected void configure(HttpSecurity http) throws Exception {
 									.usernameCollector(obriInternalCertificates)
 									.authoritiesCollector(obriInternalCertificates)
 									.build())
+							.collector(DecryptingJwtCookieCollector.builder()
```

Comment:
Jamie mentioned this to me yesterday. As you will see on my branch (feature/common-app-config-issue-210), the implementation of `CookieWebSecurityConfigurerAdapter` is the same for these applications:

- ForgerockOpenbankingAsApiApplication
- ForgerockOpenbankingRsApiApplication
- ForgerockOpenbankingRsApplication
- ForgerockOpenbankingRsStoreApplication
- ForgerockOpenbankingRsUiApplication
- SimulatorApplication

So I'm wondering if this fix needs to be applied to all of them? If so, then it might be best for me to do that on my branch, since it will only require fixing in one place.

Reply:
Yes we need to implement that in your feature also.